bundle/edu.gemini.sp.vcs/src/main/scala/edu/gemini/sp/vcs2/VcsServer.scala

```diff
@@ -92,10 +92,8 @@ class VcsServer(odb: IDBDatabaseService) { vs =>
     }
 
     (Option(p.getProgramID) \/> MissingId).liftVcs >>= { id =>
-      accessControlled(id, user) {
-        locked(p.getProgramKey, writeLock, writeUnlock) {
-          failIfExists(id, p.getProgramKey) >> putProg(odb.getFactory.copyWithNewLifespanId(p)).liftVcs
-        }
+      locked(p.getProgramKey, writeLock, writeUnlock) {
+        failIfExists(id, p.getProgramKey) >> putProg(odb.getFactory.copyWithNewLifespanId(p)).liftVcs
       }
     }
   }
```

Comment:
The `VcsServer` name may be misleading.  `VcsServer` provides low-level database and program manipulation in general.  The methods it presents are not directly accessible by remote users, who instead go through the nested `SecureVcsService`.